### ISSUE TITLE
Enhancement - Withdraw/Deposit log

### ISSUE THREAD

**Issue by jdmarlow86** [atomiclabs/hyperdex#480]
Can we include a log for withdrawal/deposit? I think the Recent Activity section of a coin in the Dashboard view would be a practical place.

**Comment by kevva** [comment]
I think this should be added in a new tab in the Trades view too.

**Comment by sindresorhus** [comment]
The plan is to have it in the Recent Activity section, but from what I can remember, @lukechilds said we would need our own Electrum server to be able to properly detect withdraw/deposit.

**Comment by sindresorhus** [comment]
@kevva It doesn't really fit to have it in the Trades view since it's not a trade, but maybe if we renamed it to History instead.

**Comment by lukechilds** [comment]
>from what I can remember, @lukechilds said we would need our own Electrum server to be able to properly detect withdraw/deposit

That's correct, this is loosely planned for the future as are a lot of other enhancements that require our own Electrum integration. But the Electrum integration will be quite a bit of work and may be totally unnecessary if mmv2 has better Electrum support so it's not something we'll be adding soon.
